README.md

```diff
@@ -45,10 +45,11 @@ This type of class stands for HTTP request parsing to a sane structure of:
  - HTTP request body
 
 ```python
+import io
 import sys
 from hotfn.http import request
 
-req = request.RawRequest(sys.stdin.read())
+req = request.RawRequest(io.fdopen(0, 'rb'))
```

Comment:
io.fdopen(sys.stdin.fileno(), 'rb')

Reply:
Okay (these are typically posix, but I suppose someone might redirect these prior to calling this)